core/templates/components/forms/schema-based-editors/schema-based-custom-editor.component.ts

```diff
@@ -90,7 +90,7 @@ implements ControlValueAccessor, Validator, AfterViewInit {
   }
 
   ngAfterViewInit(): void {
-    this.hybridForm.statusChanges.subscribe(() => {
+    this.hybridForm.statusChanges?.subscribe(() => {
```

Comment:
What does the question mark here signify, and why can't we omit it?

Reply:
Changed it now with an explanation.